Plots/MapProjections/NCL_proj_2_lg.py

```diff
@@ -0,0 +1,82 @@
+
+"""
+NCL_proj_2_lg.py
+================
+
+This script illustrates the following concepts:
+   - Drawing filled contours over a Mercator map
+   - Setting the spacing for latitude/longitude grid lines
+   - Turning off the map perimeter (boundary)
+   - Making the plot larger using viewport resources
+   - Turning off map fill
+   - Spanning part of a color map for contour fill
```

Comment:
You may want to add a new concept here that is specific to our Python implementation about our effort to introduce "inferno" as a better substitute to "rainbow"